### ISSUE TITLE
At reflow resolution "Workshops" and "Collections" control are not accessible using keyboard keys.:A11y_AzureResponsibleAIworkshop_Header_Keyboard.

### ISSUE THREAD

**Issue by PoojaNamde** [Azure/responsible-ai-hub#51]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #WCP; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #Azure Responsible AI workshop; #CelaP3_Azure Responsible AI workshop_Mar2024; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application Name: Azure Responsible AI workshop
URL: https://azure.github.io/responsible-ai-hub
Chromium Edge Version 124.0.2478.6 (Official build) dev (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: (https://azure.github.io/responsible-ai-hub/)
2. Set the reflow resolution and navigate using TAB key.
3. Observe the issue.

### Actual Results:
At reflow resolution "Workshops" and "Collections" control are not accessible using keyboard keys.

### Expected Results:
At reflow resolution "Workshops" and "Collections" controls should be accessible using keyboard keys

### User Impact:
Low vision users with keyboard will not be able to access the control and will miss the information about it.

### Attachment:
